### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by kevin-wise-infusionsoft**
I went through CAS and deleted all the unused js, css, and image files, and the security question stuff we talked about yesterday. Also renamed AppHelper to UserAccountTransformer the same as in Account Central

### CODE REVIEW

File: cas/src/main/java/org/apereo/cas/infusionsoft/config/InfusionsoftCasConfiguration.java [hunk 1]
```diff
@@ -76,8 +76,8 @@
     private UserPasswordDAO userPasswordDAO;
 
     @Bean
-    public AppHelper appHelper() {
-        return new AppHelper(infusionsoftConfigurationProperties);
+    public UserAccountTransformer appHelper() {
```
Comment (bbooth) [inline cas/src/main/java/org/apereo/cas/infusionsoft/config/InfusionsoftCasConfiguration.java]: Seems we should keep the bean name the same as the class.  This is confusing to me.

File: cas/src/main/resources/static/infusionsoft/js/password-utils.js [hunk 2]
```diff
@@ -1,29 +1,4 @@
 //Password check stuff
```
Comment (bbooth) [inline cas/src/main/resources/static/infusionsoft/js/password-utils.js]: These functions are used in the createInfusionsoftId page

File: cas/src/main/java/org/apereo/cas/infusionsoft/config/InfusionsoftCasConfiguration.java (hunk 1 above)
Comment (kevin-wise-infusionsoft) [inline cas/src/main/java/org/apereo/cas/infusionsoft/config/InfusionsoftCasConfiguration.java]: Good catch, missed that one.

File: cas/src/main/resources/static/infusionsoft/js/password-utils.js (hunk 2 above)
Comment (kevin-wise-infusionsoft) [inline cas/src/main/resources/static/infusionsoft/js/password-utils.js]: Actually, that page has copies of the functions which are slightly different.

Comment (bbooth) [inline cas/src/main/resources/static/infusionsoft/js/password-utils.js]: Are you sure everything still works on expired password?

Comment (kevin-wise-infusionsoft) [inline cas/src/main/resources/static/infusionsoft/js/password-utils.js]: Yes, I double checked that, it works

**Pull request merged**